src/ripple/overlay/Slot.h

```diff
@@ -0,0 +1,636 @@
+//------------------------------------------------------------------------------
+/*
+    This file is part of rippled: https://github.com/ripple/rippled
+    Copyright (c) 2012, 2013 Ripple Labs Inc.
+
+    Permission to use, copy, modify, and/or distribute this software for any
+    purpose  with  or without fee is hereby granted, provided that the above
+    copyright notice and this permission notice appear in all copies.
+
+    THE  SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+    WITH  REGARD  TO  THIS  SOFTWARE  INCLUDING  ALL  IMPLIED  WARRANTIES  OF
+    MERCHANTABILITY  AND  FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+    ANY  SPECIAL ,  DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+    WHATSOEVER  RESULTING  FROM  LOSS  OF USE, DATA OR PROFITS, WHETHER IN AN
+    ACTION  OF  CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+    OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+*/
+//==============================================================================
+
+#ifndef RIPPLE_OVERLAY_SLOT_H_INCLUDED
+#define RIPPLE_OVERLAY_SLOT_H_INCLUDED
+
+#include <ripple/overlay/Peer.h>
+#include <ripple/overlay/Squelch.h>
+#include <ripple/overlay/SquelchCommon.h>
+#include <ripple/protocol/PublicKey.h>
+#include <algorithm>
+#include <memory>
+#include <optional>
+#include <ripple.pb.h>
```

Comment:
This include snuck in with the wrong files. While you're there, we usually put a space between the different include groups. I.e
```
#include <ripple/overlay/SquelchCommon.h>
#include <ripple/protocol/PublicKey.h>

#include <algorithm>
#include <memory>
```